tests/cypress/e2e/features/ground_truth_jobs.js

```diff
@@ -295,14 +304,24 @@ context('Ground truth jobs', () => {
             });
         });
 
-        it('Check ground truth annotations in regular job', () => {
+        it('Check ground truth annotations in GT job and in regular job', () => {
             cy.interactMenu('Open the task');
             cy.get('.cvat-job-item').contains('a', `Job #${groundTruthJobID}`).click();
 
             groundTruthFrames.forEach((frame, index) => {
                 cy.goCheckFrameNumber(frame);
                 cy.createRectangle(groundTruthRectangles[index]);
             });
+
+            cy.changeWorkspace('Review');
+            groundTruthFrames.forEach((frame, index) => {
+                cy.goCheckFrameNumber(frame);
+                checkRectangle(groundTruthRectangles[index], true);
+                cy.get(`#cvat-objects-sidebar-state-item-${groundTruthRectangles[index].id}`)
+                    .find('.ant-dropdown-trigger').click();
```

Comment:
Instead of using `.antd` defined class, I will suggest to add CVAT class name to this element (e.g. `cvat-object-item-menu-button`) and use it. 

Also we do not need click the element, just checking that it exists and visible is enough